client/src/AjaxHelper.js

```diff
@@ -0,0 +1,125 @@
+import $ from "jquery";
+
+class AjaxHelperInstance {
+    constructor() {
+        this.url = null;
+        this._token = sessionStorage.getItem("token") ?? null;
+        // Bind
+        this.handle4xx = this.handle4xx.bind(this);
+    }
+
+    get apiUrl() {
+        return this.url;
+    }
+    get loginUrl() {
+        return this.url + "/login";
+    }
+    get logoutUrl() {
+        return this.url + "/logout";
+    }
+
+    get testLoginAsChris() {
```

Comment:
We should probably discuss what kind of simulated login mechanism we want to use in the version of this that gets merged... is there something more future-proof?